cli/args/config_file.rs

```diff
@@ -632,6 +679,8 @@ pub struct ConfigFileJson {
   pub test: Option<Value>,
   pub bench: Option<Value>,
   pub lock: Option<Value>,
+  pub include: Option<Value>,
+  pub exclude: Option<Value>,
```

Comment:
There is also https://github.com/denoland/deno/blob/main/cli/schemas/config-file.v1.json to update. Thanks for working on this!